graylog2-web-interface/src/views/components/widgets/SearchQueryExecutionInfoHelper.tsx

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import styled from 'styled-components';
+import { useCallback, useMemo, useState } from 'react';
+import numeral from 'numeral';
+
+import { Icon, Timestamp } from 'components/common';
+import { Table, Button } from 'components/bootstrap';
+import useAppSelector from 'stores/useAppSelector';
+import { selectCurrentQueryResults } from 'views/logic/slices/viewSelectors';
+import type { MessageResult, SearchTypeResult, SearchTypeResultTypes } from 'views/types';
+import type { SearchTypeIds } from 'views/logic/views/types';
+import OverlayDropdown from 'components/common/OverlayDropdown';
+
+type Props = {
+  currentWidgetMapping: SearchTypeIds,
+};
+
+const PopoverContainer = styled.div`
+  max-width: 500px;
+  padding: 5px;
+`;
+
+const QueryHelpButton = styled(Button)`
+  padding: 6px 8px;
+`;
+
+type WidgetExecutionData = {
+  total: number,
+  duration: number,
+  timestamp: string,
+  effectiveTimerange: SearchTypeResult['effective_timerange'] | MessageResult['effectiveTimerange']
+}
+const HelpPopover = ({ widgetExecutionData }: { widgetExecutionData: WidgetExecutionData}) => (
+  <PopoverContainer>
+    <p><strong>Execution Info</strong></p>
```

Comment:
You can also just use the popover title prop, this way we unify the styling.

Reply:
It would also be nice to remove the bottom margin:

![image](https://github.com/Graylog2/graylog2-server/assets/46300478/bc41772b-84ee-4770-aea5-a6589f77c45f)

Reply:
It looks like we just need to remove the table bottom margin
![image](https://github.com/Graylog2/graylog2-server/assets/46300478/db26c5e8-1221-42ba-9246-611139bcae5f)